lib/clients/github.go

```diff
@@ -167,7 +171,12 @@ func (g realGHClient) request(f func() (interface{}, *github.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = g.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
+		duration /= 1000000 // Convert nanoseconds to milliseconds
+		duration *= 1000000 // Convert back so it appears correct
```

Comment:
Just don't truncate at all. `500ms` is perfectly readable. Go gives this readability to you for free. Look again at that playground: if you specify a less legible number, e.g. `1000000000*time.Nanosecond`, you get `1s` printed.